config/locales/en/en.yml

```diff
@@ -371,16 +371,22 @@ en:
   page_title: "Page Title"
   print: "Print"
   quickbooks:
-    accounts_not_selected: 'Before you can add transactions, you need to select QuickBooks accounts on the %{settings} page.'
     authorization_failure: "You must be connected to QuickBooks before you can access transactions. Please go to %{settings} to connect."
-    modifying_transactions_not_allowed: 'Transactions are read-only because the loan is not active, transactions are in read-only mode for this loan, or both.'
     class_location_notice: "NOTE: Before you can add transactions, you need to make sure Classes and Locations (a.k.a. Divisions) are enabled in your QuickBooks company settings."
     data_reset_required: "A QuickBooks data reset is required before you can access transactions. Please go to %{settings} to perform a reset."
     department_not_set: "Please set the QB division on <a href=\"%{url}\">this loan's Madeline division</a> in order to create transactions."
     misc: "There was an error communicating with QuickBooks. An administrator has been notified. Some data may be out of date. (Error: %{msg})"
     principal_account_incompatible_with_checks: "The principal account must have type Expenses in Quickbooks in order to create check disbursements."
     service_unavailable: "QuickBooks is temporarily unavailable. Please try again later."
     negative_balance: "This transaction is not allowed because it would result in a negative balance. The total balance on this date is %{amt}."
+    read_only_html: "You can't add transactions because: %{reasons}."
+    read_only_reasons:
+      qb_not_connected_html: "QuickBooks is not connected"
+      accounts_not_selected_html: "QuickBooks accounts are not selected on the division '%{qb_division}' (see %{accounting_settings_link})"
+      division_transactions_read_only_html: "transactions are in read-only mode for the division '%{qb_division}' (see %{accounting_settings_link})"
+      department_not_set_html: "the QB department is on the division '%{current_division}' (see %{current_division_settings_link})"
```

Comment:
can i update this to 'the QB department is not set on the divisio {current division}. . . .'?

Reply:
Oops I'll fix good catch!

Reply:
Done!